content/en/_index.md

```diff
@@ -204,6 +216,11 @@ public static void exampleSnippet(String projectId, String filePath) {
     // Snippet content ...
 }
 {{< /tab >}}
+{{< tab header="Python" >}}
+# This is an example snippet for showing best practices.
+def example_snippet(project_id: str, file_path: str -> None):
```

Comment:
```suggestion
def example_snippet(project_id: str, file_path: str) -> None:
```

Reply:
Guess I should have checked my regex more carefully.